src/libcollections/vec.rs

```diff
@@ -1627,6 +1630,7 @@ impl<T> IntoIter<T> {
     #[inline]
     /// Drops all items that have not yet been moved and returns the empty vector.
     #[unstable(feature = "iter_to_vec")]
+    #[deprecated(since = "1.3.0", reason = "replaced by drain()")]
```

Comment:
Yes this isn't replaced by Drain -- in particular IntoIter allows you to return the Vec up and transport it around -- Drain doesn't. Still, this seems fairly marginal for today's std.